src/main/resources/base/ayab/translations/ayab-translation-master.tsv

```diff
@@ -31,7 +31,7 @@ StatusTab	progress	progress	Fortschritt	staða	progressione	прогресс	п
 StatusTab	Center	Centre	Zentriert	Miðja	Centra	По центру	Центр	Centre	Centrar	중앙	中心	中心
 StatusTab	Left	Left	Links	Vinstri	Sinistra	Слева	Ліворуч	Gauche	Izquierda	왼쪽	左	左
 StatusTab	Right	Right	Rechts	Hægri	Destra	Справа	Праворуч	Droite	Derecha	오른쪽	右	右
-MainWindow	All Yarns Are Beautiful	All Yarns Are Beautiful	All Yarns Are Beautiful	Allt Garn Er Fallegt	All Yarns Are Beautiful	Вся пряжа прекрасна	Уся пряжа прекрасна	Tous Les Fils Sont Beaux	Todos Los Hilos Son Hermosos	모든 실은 아름답다	すべての糸は美しいです	诸纱皆妙
+MainWindow	All Yarns Are Beautiful											
```

Comment:
I would remove the string entirely from the translation system (add `notr="true"` to the string in `main_gui.ui`), to avoid someone coming along and filling the "missing" language entries at some point.